src/index.tsx

```diff
@@ -342,26 +351,12 @@ export class Modalize<FlatListItem = any, SectionListItem = any> extends React.C
     );
   };
 
-  private onContentViewChange = (keyboardHeight?: number): void => {
+  private onContentViewChange = (): void => {
     const { adjustToContentHeight } = this.props;
-    const { contentHeight, modalHeight, headerHeight, footerHeight } = this.state;
-    const contentViewHeight = [];
-
-    if (keyboardHeight) {
-      const statusBarHeight = isIphoneX ? 48 : isIos ? 20 : StatusBarManager.HEIGHT;
-      const height =
-        screenHeight -
-        keyboardHeight -
-        headerHeight -
-        footerHeight -
-        this.handleHeight -
-        statusBarHeight;
-
-      if (contentHeight > height) {
-        contentViewHeight.push({ height });
-        this.setState({ keyboardEnableScroll: true });
-      }
-    } else if (!adjustToContentHeight) {
+    const { modalHeight, headerHeight, footerHeight } = this.state;
+    const contentViewHeight: ViewStyle[] = [];
+
+    if (!adjustToContentHeight) {
```

Comment:
When I tried to open the simple content example on Android, and press inside the input, the modal went back down, and wasn't focus on the input, and I couldn't scroll to it. For iOS it was all good